header.php

```diff
@@ -26,20 +26,18 @@
 
 	<header id="masthead" class="site-header">
 		<div class="site-branding">
-			<?php
-			the_custom_logo();
-			if ( is_front_page() && is_home() ) : ?>
```

Comment:
Here it would be more logical to just move the `?>` to the next line instead of going in and out of PHP an extra time.